frontend/dogplus-frontend/src/components/Post.tsx

```diff
@@ -52,6 +54,11 @@ export const PostCard: React.FC<PostProps> = ({ post, isLiked }) => {
     }
   };
 
+  const onComment = () => {
+    // Redirect to post detail page
+    navigate(`/post/${post.id}`);
+  }
```

Comment:
<img width="322" alt="image" src="https://github.com/DogPlus/DogPlus/assets/69573868/e6b99011-90f1-445f-8d4c-5cad4e49a7a1">

Is it supposed to show the post author name instead of id?

Reply:
Will solve this as I solve the profile picture PR